File: app/src/main/java/com/example/tcp_ble/model/LoginRequest.java
package com.example.tcp_ble.model;
import com.google.gson.annotations.SerializedName;







public class LoginRequest {
    @SerializedName("username")
    private String username;

    @SerializedName("password")
    private String password;

    @SerializedName("project_name")
    private String project_name;

    public void setUsername(String username) {
        this.username = username;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setProject_name(String project_name) {
        this.project_name = project_name;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public String getProject_name() {
        return project_name;
    }


    // Add getters and setters for other fields as needed
}

File: app/src/main/java/com/apogee/registration/model/LoginRequest.kt
package com.apogee.registration.model


import com.google.gson.Gson
import com.google.gson.JsonObject
import com.google.gson.annotations.SerializedName


data class LoginRequest(
    @SerializedName("password")
    val password: String,
    @SerializedName("project_name")
    val projectName: String,
    @SerializedName("username")
    val username: String
){
    fun setJsonObject(): JsonObject {
        val json=JsonObject()
        json.add("password",Gson().toJsonTree(this.password))
        json.add("project_name",Gson().toJsonTree(this.projectName))
        json.add("username",Gson().toJsonTree(this.username))
        return json
    }
}